full-stack-tests/presentation/src/backend/PresentationManager.test.ts

```diff
@@ -87,7 +87,7 @@ describe("PresentationManager", () => {
     });
 
     async function getAreaDisplayValue(unitSystem: UnitSystemKey, defaultFormats?: { [phenomenon: string]: UnitSystemFormat }): Promise<DisplayValue> {
-      return using(new PresentationManager({ defaultFormats, defaultLocale: "en-PSEUDO" }), async (manager) => {
+      return using(new PresentationManager({ defaultFormats, defaultLocale: "en" }), async (manager) => {
```

Comment:
So the psuedo-localization changes probably warrant a larger discussion. We get lots of complaints about strings not being localized/looking weird because when users run an imodeljs powered app in development, it defaulted to en-pseudo. Late in 2.x we stopped delivering psuedo-localized files altogether in core. In the big i18n refactor pr that just went through we defaulted to using en in development, and plan to have a webpack plugin to do psuedo-localization for us. Due to this change, i updated most pkgs to not run the pusedo-localization script or copying it over it into the lib output deliverable. 

I can revert this change, and continue to have presentation do psuedo-localization like before, but making sure we don't deliver it to end users if you'd really like.

Reply:
I don't mind us changing the default from "en-pseudo" to "en" and I agree that we probably should not deliver "en-pseudo" with our packages. But I would still like us to use "en-pseudo" in tests just to ensure we localize our strings.